tests/dragonfly/replication_test.py

```diff
@@ -2763,7 +2763,7 @@ async def get_memory(client, field):
 @pytest.mark.exclude_epoll
 @pytest.mark.parametrize(
     "element_size, elements_number",
-    [(16, 30000), (30000, 16)],
+    [(16, 20000), (30000, 16)],
```

Comment:
we also have other places were we use DebugPopulateSeeder and we excluded the stream type in this test f.e the test_cluster_migration_huge_container
can you add the stream to this tests make sure they are passing